src/Tables/index.ts

```diff
@@ -26,11 +27,14 @@ export interface ITableColumnConfig {
   hasCopyButton?: boolean
   width?: number // TODO: This!
 }
+
+export type IStatus = 'caution' | 'danger' | 'folder' | 'good' | 'neutral';
```

Comment:
The only two places I found these values are related to the table.

<img width="1429" alt="Screen Shot 2021-03-18 at 22 25 23" src="https://user-images.githubusercontent.com/10409078/111633431-d6e7e580-8838-11eb-8d6f-bb4a9b0d8a69.png">

Both Components that required have access to it.

I have updated the code with this relation in any case to avoid updating one without updating the other.

<img width="1526" alt="Screen Shot 2021-03-18 at 22 24 07" src="https://user-images.githubusercontent.com/10409078/111633274-a902a100-8838-11eb-8176-d183d289654a.png">

However if you prefer it to be even higher, I can update it and put it in the general index.